src/main/java/com/revature/services/QuestionLogicImpl.java

```diff
@@ -19,10 +21,33 @@
 	@Autowired
 	private QuestionRepository dao;
 
+    @Autowired
+    private JsonValidation validation;
+
+    @PersistenceContext
+    private EntityManager entityManager;
+
+	
+//CREATE-----------------------------------
+	@Override
+	@Transactional
+	public QuestionPool createQuestion(QuestionPool question) {
+		
+		validation.validateQuestionPoolFields(question);
+		
+		question.setUseCount(0);
+        question.setDateLastUsed(null);
+		question = dao.saveAndFlush(question);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Introduce a new variable instead of reusing the parameter "question". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1226)